service/provider/src/main/java/gov/va/vro/service/provider/bip/service/BipClaimService.java

```diff
@@ -169,6 +170,9 @@ public MasProcessingObject completeProcessing(MasProcessingObject payload) {
    */
   public FetchPdfResponse uploadPdf(FetchPdfResponse pdfResponse) {
     log.info("Uploading pdf for claim {}...", pdfResponse.getClaimSubmissionId());
+    if (pdfResponse.getPdfData() == null) {
+      throw new MasException("PDF Response does not contain any data");
+    }
     String filename = String.format("temp_evidence-%s.pdf", pdfResponse.getClaimSubmissionId());
```

Comment:
Should MasException be specified with throws? Similarly, should the BipException thrown by bipApiService.uploadEvidence(...) be caught or thrown?

Reply:
@wlin2019 good points. I changed to BipException and declared it with "throws"